bin/df/df.c

```diff
@@ -131,8 +131,10 @@ main(int argc, char *argv[])
 			 * be set to 1024. Thus, if this occurs, simply break
```

Comment:
I think the last sentence `Thus, if this occurs, simply break ather than clobbering the old blocksize.` should be removed.